build/vsts_build.yaml

```diff
@@ -183,6 +183,20 @@ jobs:
       optimizationInputsDropNamePrefix: OptimizationInputs/$(System.TeamProject)/$(Build.Repository.Name)
     condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
 
+  - task: PowerShell@1
+    displayName: "Install dotnet-format"
+    inputs:
+      scriptType: "inlineScript"
+      inlineScript: |
+        dotnet tool install --global dotnet-format
```

Comment:
`dotnet tool install` returns exit code 1 if the tool is already installed. Therefore, if this runs on the same agent more than once the CI build will always fail.

Reply:
Returning exit code 1 in this scenario seems like an interesting design choice... ... ... 😅

Reply:
what. seriously?

Reply:
If this approach otherwise works, there's a flag we can set to tell AzDO to treat the error as a warning, so it doesn't fail the build. It will mean we can't tell the difference between a real failure and already-installed, but it is what it is.